wandb/proto/wandb_internal.proto

```diff
@@ -1383,6 +1383,11 @@ enum ServerFeature {
   ARTIFACT_TAGS = 1;
   CLIENT_IDS = 2;
   ARTIFACT_REGISTRY_SEARCH = 3;
+  AUTOMATION_EVENT_RUN_METRIC = 4;
```

Comment:
Could you please add docstring explaining the features?

Reply:
sure!

Reply:
Thanks! Also asked someone on the sdk team to add for the other features 😉

Reply:
Done!